### PR TITLE
fix: issue #387 Out of range value for column 'total_sent'

### PR DESCRIPTION
**Pull request by waqarRns**

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -949,6 +949,8 @@ export class LedgerStorage extends Storages {
                     .then(async (rows: any[]) => {
                         let senders = [];
                         let receivers = [];
+                        let total_recieved = JSBI.BigInt(0);
```
Comment (Geod24) [inline src/modules/storage/LedgerStorage.ts]:
```suggestion
                        let total_received = JSBI.BigInt(0);
```

**Comment by TrustHenry** [comment]
```
agora_1     | 2021-07-26 08:33:22,317 Info [boa1xrval7gwhjz4k9raqukcnv2n4rl4fxt74m2y9eay6l5mqdf4gntnzhhscrh] - Accepted transaction: Inputs (1): 0x80d1...7a63:0x5963...d49a
toolsdev_1  | AutoSend: Status = true; Hash = 0xdb5bab6ea84412960209b57fa1630a5216af9a4b8eedf7c47cee486e2b23af3eddedfbe9dc6c66d6d197f1c8b1d1ab4e8d10de7da6a52edd0e1faa4c177d88f2
stoa_1      | (node:26) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 231)
stoa_1      | (node:26) UnhandledPromiseRejectionWarning: Error: Out of range value for column 'total_received' at row 1
stoa_1      |     at Packet.asError (/stoa/bin/node_modules/mysql2/lib/packets/packet.js:712:17)
stoa_1      |     at Query.execute (/stoa/bin/node_modules/mysql2/lib/commands/command.js:28:26)
stoa_1      |     at Connection.handlePacket (/stoa/bin/node_modules/mysql2/lib/connection.js:425:32)
stoa_1      |     at PacketParser.onPacket (/stoa/bin/node_modules/mysql2/lib/connection.js:75:12)
stoa_1      |     at PacketParser.executeStart (/stoa/bin/node_modules/mysql2/lib/packet_parser.js:75:16)
stoa_1      |     at Socket.<anonymous> (/stoa/bin/node_modules/mysql2/lib/connection.js:82:25)
stoa_1      |     at Socket.emit (events.js:315:20)
stoa_1      |     at Socket.EventEmitter.emit (domain.js:467:12)
stoa_1      |     at addChunk (internal/streams/readable.js:309:12)
stoa_1      |     at readableAddChunk (internal/streams/readable.js:284:9)
stoa_1      | (node:26) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 232)
```

**Comment by TrustHenry** [comment]
Error: Out of range value for column 'total_received' at row 1
`total_received` and `total_sent` <- Out of range due to cumulative value.

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -300,8 +300,8 @@ export class LedgerStorage extends Storages {
         CREATE TABLE IF NOT EXISTS accounts(
             address          TEXT,
             tx_count         INTEGER,
-            total_received   BIGINT(20) UNSIGNED NOT NULL,
-            total_sent       BIGINT(20) UNSIGNED NOT NULL,
+            total_received   BIGINT(24) UNSIGNED NOT NULL,
+            total_sent       BIGINT(24) UNSIGNED NOT NULL,
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:304]: https://github.com/bosagora/stoa/blob/v0.x.x/docs/Database_Schema.md

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**